drivers/sensor/pmw3360/pmw3360.c

```diff
@@ -135,8 +144,8 @@ struct pmw3360_data {
 static const struct spi_config spi_cfg = {
 	.operation = SPI_WORD_SET(8) | SPI_TRANSFER_MSB |
 		     SPI_MODE_CPOL | SPI_MODE_CPHA,
-	.frequency = CONFIG_DESKTOP_SPI_FREQ_HZ,
-	.slave = 0,
+	.frequency = DT_INST_0_PIXART_PMW3360_SPI_MAX_FREQUENCY,
```

Comment:
consider using define's here too

Reply:
I prefer to leave it. I used macros in other place to make code lines shorter. Here it does not mater much really.